src/mumble/MainWindow.cpp

```diff
@@ -200,6 +202,27 @@ MainWindow::MainWindow(QWidget *p)
 	QAccessible::installFactory(AccessibleSlider::semanticSliderFactory);
 }
 
+// Loading a state that was stored by a different version of Qt can lead to a crash.
+// This function calculates the state version based on Qt's version and MainWindow.ui's hash (provided through CMake).
+// That way we also avoid potentially causing bugs/glitches when there are changes to MainWindow's widgets.
+int MainWindow::stateVersion() {
+	constexpr size_t hashLength = sizeof(MAINWINDOW_UI_HASH);
```

Comment:
Does `sizeof` yield the length of the string literal? Normally it only gives the byte size if the underlying type, which in this case would be `char *` I.e. 8 bytes on most platforms.

Reply:
It does for preprocessor defines.

Reply:
Wild ^^